lib/kms/file/backend.js

```diff
@@ -0,0 +1,80 @@
+import common from '../common';
+
+export const backend = {
+    /*
+     * Target implementation will be async in some aspects. let mimic it
+     */
+
+    /*
+     * cb(err, masterKeyId: string)
+     */
```

Comment:
Add comment: 
Using createDataKey here for purposes of createBucketKeyMem so that we do not need a separate function.

or just use random bytes.